src/mesh/mesh-tenant.model.ts

```diff
@@ -1,6 +1,7 @@
 import { Account } from "../aws/aws.model.ts";
 import { Subscription } from "../azure/azure.model.ts";
 import { Project } from "../gcp/gcp.model.ts";
+import { MeshTenantRoleAssignment } from "./mesh-iam-model.ts";
 
 export interface MeshTag {
```

Comment:
d: Could it make sense to also include MeshTags into MeshTenants on the medium term?

Reply:
I'm not sure what you mean?